packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -478,7 +478,7 @@ class CupertinoIcons {
   ///  * [refresh_bold], which is similar, but rotated 45 degrees clockwise and thicker.
   static const IconData refresh_thick = IconData(0xf3a8, fontFamily: iconFont, fontPackage: iconFontPackage);
 
-  /// An arrow on a circular path with its end pointing at its start.
+  /// <i class='cupertino-icons md-36'>arrow_counterclockwise</i> &#x2014; cupertino icon for an arrow on a circular path with its end pointing at its start.
```

Comment:
Should this be something else since arrow_counterclockwise is already used above?